tutorials/k8s-kapsule-multi-az/index.mdx

```diff
@@ -0,0 +1,614 @@
+</Message>
+
+The domain used for this tutorial can be of type `internal` when bought with your Scaleway account, or of the `external` type if you want to re-use a domain bought with an external DNS registrar. If you choose the `external` method, you can follow the [How to add an external domain to DNS](https://www.scaleway.com/en/docs/network/domains-and-dns/how-to/add-external-domain/) documentation. If you don't want to delegate all your domain to your Scaleway account, import your domain, but do not change the name servers in your registrar. Then you can create a "DNS zone" in Scaleway on your external domain, and declare two NS records in your registrar matching this zone. For example, with a domain "your-domain.tld" imported in Scaleway, you can create a DNS zone "scw", and delegate this zone in your registrar by adding those two records:
+    ```
+    scw 600 IN NS ns0.dom.scw.cloud.
+    scw 600 IN NS ns1.dom.scw.cloud.
+    ```
+The zone "scw.your-domain.tld" is now managed in your Scaleway account.
+
+1. Create a new file `ingress-nginx.tf` to store the deployment configuration of your ingress controller. The first step is to configure `kubectl` and `helm` providers to use the host and credentials generated by the `scaleway_k8s_cluster` resource when we created the Kapsule cluster.
+    ```
+    provider "kubernetes" {
+        host                   = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].host
+        token                  = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+        cluster_ca_certificate = base64decode(scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate)
+    }
+
+    provider "kubectl" {
+        host                   = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].host
+        token                  = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+        cluster_ca_certificate = base64decode(scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate)
+        load_config_file       = false
+    }
+
+    provider "helm" {
+        kubernetes {
+            host                   = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].host
+            token                  = scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].token
+            cluster_ca_certificate = base64decode(scaleway_k8s_cluster.kapsule_multi_az.kubeconfig[0].cluster_ca_certificate)
+        }
+    }
+    ```
+
+2. To configure the distribution of the pods on the cluster we will use labels set on nodes. To get the label list of a node, you can use the `kubectl` command below. You can see that multiple labels, sets by the `scaleway-cloud-controller-manager`, contain information about the `region` and `zone` where the node is deployed. The label `topology.kubernetes.io/zone`, at the end of the list, is particularly interesting in your case as it will allow us to know on which availability zone a pod will be deployed.
```

Comment:
```suggestion
2. Use the `kubectl` command below to get the label list of a node. We will use label sets on nodes to configure the distribution of the pods on the cluster. You can see that multiple labels, sets by the `scaleway-cloud-controller-manager`, contain information about the `region` and `zone` where the node is deployed. The label `topology.kubernetes.io/zone`, at the end of the list, is particularly interesting in your case as it will allow us to know on which availability zone a pod will be deployed.
```